examples/reference/elements/matplotlib/Text.ipynb

```diff
@@ -37,9 +37,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "%%opts Curve (color='#D3D3D3')\n",
     "xs = np.linspace(-5,5,100)\n",
-    "hv.Curve((xs,-(xs-2)**2))  * hv.Text(0, -30, 'Quadratic Curve')"
+    "hv.Curve((xs,-(xs-2)**2)).opts(color='#D3D3D3') * hv.Text(0, -30, 'Quadratic Curve')"
```

Comment:
Either:

```python
curve = hv.Curve((xs,-(xs-2)**2))
text = hv.Text(0, -30, 'Quadratic Curve')
curve.opts(color='#D3D3D3') * text

Or:

```python
curve = hv.Curve((xs,-(xs-2)**2)).opts(color='#D3D3D3')
text = hv.Text(0, -30, 'Quadratic Curve')
curve * text
```


```python
(hv.Curve((xs,-(xs-2)**2)) * hv.Text(0, -30, 'Quadratic Curve')).opts(
    opts.Curve(color='#D3D3D3'))
```

Reply:
Styling only one element I still prefer ``curve.opts(color='#D3D3D3') * text``

Reply:
In a comment below I point out that maybe we should be using `opts.defaults` here instead.